src/audio/mic_privacy_manager/mic_privacy_manager.c

```diff
@@ -0,0 +1,234 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2025 Intel Corporation. All rights reserved.
+
+#include "sof/audio/mic_privacy_manager.h"
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/mic_privacy.h>
+#include <zephyr/logging/log.h>
+#include <sof/audio/audio_stream.h>
+#include <sof/audio/buffer.h>
+#include <sof/audio/mic_privacy_manager.h>
+
+const struct device *mic_priv_dev;
+
+static struct mic_privacy_api_funcs *mic_privacy_api;
+static enum mic_privacy_policy mic_privacy_policy;
+
+#define LOG_DOMAIN mic_priv
+
+LOG_MODULE_REGISTER(LOG_DOMAIN);
+
+int mic_privacy_manager_init(void)
```

Comment:
this file is Intel specific, can we emphasize it with file name and external API names?